generators/app/templates/client/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title><%= _.kebabCase(appName) %></title>
```

Comment:
Don't you think kebabCase for a default title is a bad choice? Kebab Case is simply not user-friendly. It's good for app names, but not very good for the app's title on the browser.

A project named `observatory` would be kept as `observatory` and a `star_observatory` would be transformed into `star-observatory.

As it was before, it was transforming it to Pascal Case, so `observatory` would become `Observatory` and `star_observatory` would become `Star Observatory`.